### PR TITLE
Add version 0.0.15

### PR DESCRIPTION
**Pull request by functorism**

### CODE REVIEW

File: purerl/0.0.15.nix [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import ./mkPurerlDerivation.nix ( rec{
+  inherit pkgs;
+
+  version = "0.0.15";
+
+  src =
+    if pkgs.stdenv.isDarwin then
+      pkgs.fetchurl {
+        url = "https://github.com/purerl/purerl/releases/download/v${version}}/macos.tar.gz";
```
Comment (nwolverson) [inline purerl/0.0.15.nix]: This doesn't look right

File: purerl/0.0.15.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        sha256 = "0xa2wj9ayg8n2c1i8059r60byryz4j8sjj2aw1x5akb33wdd78i9";
+      }
+    else
+      pkgs.fetchurl {
+        url = "https://github.com/purerl/purerl/releases/download/v${version}}/linux64.tar.gz";
```
Comment (nwolverson) [inline purerl/0.0.15.nix]: Extra `}` here too

File: purerl/0.0.15.nix (hunk 1 above)
Comment (nwolverson) [inline purerl/0.0.15.nix]:
```suggestion
        url = "https://github.com/purerl/purerl/releases/download/v${version}/macos.tar.gz";
```

File: purerl/0.0.15.nix (hunk 2 above)
Comment (nwolverson) [inline purerl/0.0.15.nix]:
```suggestion
        url = "https://github.com/purerl/purerl/releases/download/v${version}/linux64.tar.gz";
```

**Review by nwolverson — CHANGES_REQUESTED** [review]

File: purerl/0.0.15.nix (hunk 1 above)
Comment (functorism) [reply purerl/0.0.15.nix]: Got burned by nix-prefetch : [
fixed now

**Pull request merged**